bin/release_tag.rb

```diff
@@ -18,7 +18,12 @@
 review = StringIO.new
 post_review = StringIO.new
 
-ManageIQ::Release.repos_for(opts).each do |repo|
+# Move manageiq repo to the end of the list.  The rake release script on manageiq
+#   depends on all of the other repos running their rake release scripts first.
+repos = ManageIQ::Release.repos_for(opts)
+repos = repos.partition { |r| r.github_repo != "ManageIQ/manageiq" }.flatten
```

Comment:
If it is easier for you to read, you could put this into a new method `ordered_repos_for`